components/input/gpio/gpio_test.go

```diff
@@ -63,7 +63,7 @@ func setup(t *testing.T) *setupResult {
 			"interrupt1": {
 				Control:    input.ButtonNorth,
 				Invert:     false,
-				DebounceMs: 0,
+				DebounceMs: 20,
```

Comment:
Changed from the default 5ms

Reply:
Probably fine. Though the original intent was to also check that the default was applied, when the actual value was zero (e.g. when not set in json.)

Reply:
I'll look into adding another test to check that defaults are set in another pr